src/main/java/de/qabel/desktop/ui/accounting/wizard/WizardController.java

```diff
@@ -123,20 +156,42 @@ private void addListeners() {
                     phoneInput.getStyleClass().removeAll("error-textfield");
                 } else {
                     phoneInput.getStyleClass().add("error-textfield");
-                    phoneLabel.setText("");
                 }
             }
         });
     }
 
+    private BooleanBinding publish;
+
     private void validateButtons() {
         backButton.visibleProperty().bind(currentStepIndex.greaterThan(0));
         finishButton.visibleProperty().bind(currentStepIndex.isEqualTo(steps.size() - 1));
         emailLabel.visibleProperty().bind(emailInput.textProperty().isEmpty().not());
-        nextButton.disableProperty().bind(aliasInput.textProperty().isEmpty());
         nextButton.visibleProperty().bind(currentStepIndex.lessThan(3));
         phoneLabel.visibleProperty().bind(phoneInput.textProperty().isEmpty().not());
         aliasLabel.visibleProperty().bind(aliasInput.textProperty().isEmpty().not());
+
+
+        BooleanBinding hasAlias = aliasInput.textProperty().isNotEmpty();
+        BooleanBinding hasEmail = emailInput.textProperty().isNotEmpty();
+        BooleanBinding hasPhone = phoneInput.textProperty().isNotEmpty();
+        BooleanBinding emailValid = createBooleanBinding(
+            () -> emailInput.getText().contains("@"),
+            emailInput.textProperty()
+        );
+        BooleanBinding phoneValid = createBooleanBinding(
+            () -> validatePhone(phoneInput.getText()),
+            phoneInput.textProperty()
+        );
+        nextButton.disableProperty().bind(
+            hasAlias.not().or(
+                currentStepIndex.isEqualTo(1).and(hasEmail).and(emailValid.not())
+            ).or(
+                currentStepIndex.isEqualTo(2).and(hasPhone).and(phoneValid.not())
+            )
+        );
```

Comment:
Disable next button if:
 - on second (1) step and there is an invalid email or
 - on third (2) step and there is an invalid phone number